app/src/main/java/com/github/se/polyfit/model/post/Post.kt

```diff
@@ -20,13 +21,9 @@ data class Post(
     override var location: Location,
     override var meal: Meal,
     override var createdAt: LocalDate,
-    var listOfImages: List<Bitmap> = emptyList()
+    var listOfImages: List<Bitmap> = emptyList(),
+    var listOfURLs: List<StorageReference> = emptyList()
```

Comment:
Why is storing both the image (the bitmap) and the storage reference needed? Is this for a future implementation or for backwards compatibility?

Also, it might be a better idea to use val and mutable lists instead of var. This would make adding new elements to the lists easier, as there would not be a need to create a new list each time.

Reply:
listOfImages where the list of the pictures the user wants to add to the post, first I tried to fetch locally the pictures, it's doable but nobody in the industry do it like this but keep the picture references and display them whit Glide.

if we would have a cleaner code we should need to review and rewrite a big chunk of the code